### PR TITLE
more terra to paloma

### PR DESCRIPTION
**Pull request by rootedbox**

### CODE REVIEW

File: integration-tests/cancelUpgradeProposal.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ const client = new LCDClient({
   gasPrices: { uusd: 0.38 },
 });
 
-// LocalTerra test1 terra1x46rqay4d3cssq8gxxvqz8xt6nwlz4td20k38v
+// LocalPaloma test1 terra1x46rqay4d3cssq8gxxvqz8xt6nwlz4td20k38v
```
Comment (taariq) [inline integration-tests/cancelUpgradeProposal.ts]: This is still a Terra address

File: integration-tests/submitProposal.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ const client = new LCDClient({
   URL: 'https://bombay-lcd.terra.dev',
 });
 
-// LocalTerra test1 terra1x46rqay4d3cssq8gxxvqz8xt6nwlz4td20k38v
+// LocalPaloma test1 terra1x46rqay4d3cssq8gxxvqz8xt6nwlz4td20k38v
```
Comment (taariq) [inline integration-tests/submitProposal.ts]: Another Terra address

**Comment by measure-fi** [comment]
```
☞ rg -i 'terra|luna' | wc -l
18290
```
😅

**Comment by measure-fi** [comment]
And actually even if we just look within the diff

```
☞ git log -1 -p | rg -i 'terra|luna'
```

there are still 501 lines that need fixing.

**Comment by rootedbox** [comment]
We are still using legacy.proto terra.proto libraries.. so they will cause terra to be found. I'm just trying to update the README and the tests.

File: integration-tests/submitProposal.ts (hunk 2 above)
Comment (verabehr) [reply integration-tests/submitProposal.ts]: ... and above there is actually a reference to bombay-lcd.terra.dev 

This needs more care than just a simple find and replace

**Pull request merged**